matomo/Jenkinsfile

```diff
@@ -1,5 +1,10 @@
 pipeline {
     agent none
+    environment {
+            COMPONENT_NAME = 'Matomo Analytics'
+            COMPONENT_HOME = 'matomo'
+            BUILD_TRIGGER_INCLUDES = '^matomo/'
```

Comment:
@cvarjao I think this would be a great feature for the bcdk to have this included as apart of the pipeline.